nautobot/dcim/forms.py

```diff
@@ -2508,6 +2508,27 @@ class InterfaceFilterForm(DeviceComponentFilterForm):
 
 
 class InterfaceForm(NautobotModelForm, InterfaceCommonForm):
+    parent_interface = DynamicModelChoiceField(
+        queryset=Interface.objects.all(),
+        required=False,
+        label="Parent interface",
+        query_params={
+            "kind": "physical",
+        },
+    )
+    bridge = DynamicModelChoiceField(
+        queryset=Interface.objects.all(),
+        required=False,
+        label="Bridge interface",
+    )
+    lag = DynamicModelChoiceField(
+        queryset=Interface.objects.all(),
+        required=False,
+        label="LAG interface",
+        query_params={
+            "type": InterfaceTypeChoices.TYPE_LAG,
+        },
+    )
```

Comment:
Please add `help_text` to these fields since it won't be inherited from the model as they were overloaded on this form.